components/portal_players/commons/portal_players.lua

```diff
@@ -284,6 +285,18 @@ function PortalPlayers:row(player, isPlayer)
 	return row
 end
 
+---@param status string?
+---@param banned string?
+---@return string?
+function PortalPlayers._getStatusBackground(status, banned)
+	if status == STATUS_INACTIVE then
+		status = Logic.emptyOr(Logic.readBoolOrNil(banned), Logic.isNotEmpty(banned))
+			and 'banned' or nil
```

Comment:
Otherwise the inactive status is overridden with nil
```suggestion
	if Logic.emptyOr(Logic.readBoolOrNil(banned), Logic.isNotEmpty(banned)) then
		status = 'banned'
```

Reply:
```suggestion
	if status == STATUS_INACTIVE then
		status = Logic.emptyOr(Logic.readBoolOrNil(banned), Logic.isNotEmpty(banned))
			and 'banned' or status
```

Reply:
What's the reason to make this more complicated then it needs to be?

Reply:
you could have a status ~= inactive while still having banned being set

Reply:
I don't think that should be a thing, what's the point of setting as banned then?

Reply:
i am thinking of cases where one is banned and retired
imo in that case retired is what should be shown

Reply:
Is it? Likely the retirement is due to the ban, no?

Reply:
i (personally) think the being retired is the more important information